packages/kbn-journeys/journey/journey_screenshots.ts

```diff
@@ -92,7 +92,7 @@ export class JourneyScreenshots {
     await this.lock(async () => {
       const filename = FtrScreenshotFilename.create(`${step.index}-${step.name}-failure`);
       const fullscreenFilename = FtrScreenshotFilename.create(
-        `${step.index}-${step.name}-failure-fullscreen`
+        `${step.index}-${step.name.replace(/\s/g, '-')}-failure-fullscreen`
```

Comment:
@dmlemeshko this `.replace()` should really be done inside the `FtrScreenshotFilename.create()` function.